bittensor/__init__.py

```diff
@@ -337,16 +366,15 @@ def debug(on: bool = True):
 
 from .subnets import SubnetsAPI as SubnetsAPI
 
+# TODO: Do we still need this? This is a really bad way to do this. We currently re initializing the config for every
```

Comment:
@LogrisTheBard please review / comment on TODO

Reply:
I have no attachment to this way of doing things but I think changing this would raise an eyebrow during the code review that we just don't need to explain as part of _this_ review cycle which is already quite large.